README.md

```diff
@@ -40,85 +57,97 @@ You would have to change a few things (see below, build from scratch)
 
 From your shell or command line:
 
-   ```Shell
-   git clone https://github.com/Azure-Samples/active-directory-dotnet-native-uwp-v2.git
-   ```
+```Shell
+git clone https://github.com/Azure-Samples/active-directory-dotnet-native-uwp-v2.git
+```
 
-or download and exact the repository .zip file.
+or download and extract the repository .zip file.
 
-> Given that the name of the sample is pretty long, and so are the name of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
+> Given that the names of the referenced NuGet packages are quiet long, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
 
 ### Step 2:  Register the sample application with your Azure Active Directory tenant
 
 There is one project in this sample. To register it, you can:
 
 - either follow the steps [Step 2: Register the sample with your Azure Active Directory tenant](#step-2-register-the-sample-with-your-azure-active-directory-tenant) and [Step 3:  Configure the sample to use your Azure AD tenant](#choose-the-azure-ad-tenant-where-you-want-to-create-your-applications)
 - or use PowerShell scripts that:
-  - **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you
+  - **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you. Note that this works for Visual Studio only.
   - modify the Visual Studio projects' configuration files.
 
-If you want to use this automation:
-1. On Windows run PowerShell and navigate to the root of the cloned directory
+<details>
+  <summary>Expand this section if you want to use this automation:</summary>
+
+1. On Windows, run PowerShell and navigate to the root of the cloned directory
 1. In PowerShell run:
+
    ```PowerShell
    Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
    ```
-1. Run the script to create your Azure AD application and configure the code of the sample application accordinly. 
+
+1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+1. In PowerShell run:
+
    ```PowerShell
-   .\AppCreationScripts\Configure.ps1
+   cd .\AppCreationScripts\
+   .\Configure.ps1
    ```
+
    > Other ways of running the scripts are described in [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md)
+   > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+1. Open the Visual Studio solution and click start to run the code.
 
-1. Open the Visual Studio solution and click start
+</details>
 
-If ou don't want to use this automation, follow the steps below
+Follow the steps below to manually walk through the steps to register and configure the applications.

```

Comment:
```suggestion
<details>
  <summary>Expand this section to manually walk through the steps to register and configure the applications:</summary>
```

Reply:
As per current approach to create Readme files, this section is displayed like this.
Thanks!

Reply:
Thanks for letting me know. Is there a guideline document you can point me to so I can read it?

Reply:
These steps are generated by **Code Generator**. 
Thanks!